src/js/conduit/Router.test.js

```diff
@@ -2,6 +2,124 @@ import '@babel/polyfill';
 
 import Router from './Router';
 
-test('adds 1 + 2 to equal 3', async () => {
-  expect(1 + 2).toBe(3);
+describe('Router', () => {
+  describe('export value', () => {
```

Comment:
@brainthinks review naming conventions for tests - tests can be more english than robot.

Reply:
see https://github.com/facebook/react/blob/f625fce8578cc77113d9da8ffd02032a60477e6a/packages/react-dom/src/events/__tests__/SyntheticKeyboardEvent-test.js#L35 for example

Reply:
i have re-written all of the "describe" and "it" strings